x/dex/keeper/liquidity.go

```diff
@@ -107,3 +127,14 @@ func (k Keeper) SaveLiquidity(sdkCtx sdk.Context, liquidityI types.Liquidity) {
 		panic("Invalid liquidity type")
 	}
 }
+
+var MaxTrueMakerSpread = math_utils.MustNewPrecDecFromStr("0.005")
```

Comment:
Would it be better to move this magic constant to the module params?

Reply:
👍